### PR TITLE
Log WMCO timestamps in RFC3339 format

### PR DESCRIPTION
**Pull request by sebsoto**
Makes timestamps human readable for easier use

### CODE REVIEW

**Comment by sebsoto** [comment]
Example logs with new timestamp format: 
```
2022-08-08T17:53:39Z	INFO	Starting EventSource	{"controller": "certificatesigningrequest", "controllerGroup": "certificates.k8s.io", "controllerKind": "CertificateSigningRequest", "source": "kind source: *v1.Secret"}
2022-08-08T17:53:39Z	INFO	Starting EventSource	{"controller": "certificatesigningrequest", "controllerGroup": "certificates.k8s.io", "controllerKind": "CertificateSigningRequest", "source": "kind source: *v1.Secret"}
```

**Review by jrvaldes — APPROVED** [review]
/lgtm

**Comment by mansikulkarni96** [comment]
/lgtm 
Thanks for working on the improvement @sebsoto

File: go.mod [hunk 1]
```diff
@@ -94,7 +95,6 @@ require (
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
 	go.uber.org/atomic v1.7.0 // indirect
 	go.uber.org/multierr v1.6.0 // indirect
-	go.uber.org/zap v1.19.1 // indirect
```
Comment (aravindhp) [inline go.mod:97]: If this was done due to a go mod update, please add the command you ran to the commit message.

**Review by aravindhp — CHANGES_REQUESTED** [review]
Nice improvement, @sebsoto.

**Review by aravindhp — APPROVED** [review]

**Pull request merged**